app/components/Main.js

```diff
@@ -1,106 +1,80 @@
+//Standard imports
 var React = require('react');
-var Link = require("react-router").Link; 
+
+var router = require('react-router-dom');
+var Link = router.Link;
+var Route = router.Route;
+
+//codebase imports
+var helpers = require('./utils/helpers');
+
+var Dashboard = require('./children/dashboard');
+var PodLanding = require('./children/podLanding');
+
 //subcomponents go here, not sure what they'll be called yet
 // var ??? = require("./???");
-var helpers = require('../utils/helpers');
+
+
 var Main = React.createClass({
 
+  getInitialState: function() {
+    return { 'keyword': this.keyword };
```

Comment:
this may be something we take out for now. It is supposed to enable us to search pods by a keyword